systers_portal/templates/community/page.html

```diff
@@ -0,0 +1,15 @@
+{% extends "community/base.html" %}
+{% block title %}
+- {{ community }}
+{% endblock %}
+{% block community_page_content %}
+<div class="blog-container">
+<div class="blog-entry">
+<h3 class="title"><a href="#">{{ page.title }}</a></h3>
+<p class="meta">{{ page.date_modified }} | <a
```

Comment:
Remove the modified field. Community Page isn't supposed to look like a blogpost. It's more of a stable page. Examples of Community Pages: Information, General, How to reach us, etc.